books/dbt-tutorial/chapter1.md

```
@@ -0,0 +1,88 @@
+title: "はじめに"
+---
+
+
+## はじめに
+
+このチュートリアルはすべてDocker環境で行うためローカル環境を汚染しません。
+最初のベースとなるファイルのみレポジトリに配置されているので、それらをどんどんと変換していくのがこのチュートリアルの基本的な流れとなっています。
+
+また、dbtに用いるOLAPDBとしてDuckDBを使用しています。
+今回はDuckDBの中身まで言及はしませんが、SQLiteのOLAPDB版とざっくり捉えていただいて構いません。
+SQLiteのようにサーバを立ち上げなくてもファイルベースで管理できるとても優れたDBです。
```

markdownlint-cli2 (0.17.2)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Above

(MD022, blanks-around-headings)